ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -195,3 +195,8 @@ tests:
   steps:
     cluster_profile: aws
     workflow: origin-e2e
+- as: e2e-baremetalds
+  commands: ""
```

Comment:
@bbguimaraes we should not require this :|

Reply:
It's not required.

Reply:
@stevekuznetsov @bbguimaraes are you referring to the empty `commands: ""`?